### DISCUSSION TITLE
Feature request: multiple bids on the same order from a single provider

### DISCUSSION THREAD

**Discussion by shimpa1** [akash-network/community#516] [AI - Artificial Intelligence]
hi folks,

This is relevant to providers with more than one GPU model in their portfolio.

Right now if a deployer does not specify the GPU model in the SDL, a provider pod will place a bid using the catchall entry in the provider.yaml file.
Example:
```price_target_gpu_mappings: "h100=1088,rtxa6000=358,*=1088"```
The bid will be placed using the ```*=1088``` entry and the ```rtxa6000=358``` entry will not get used ever unless explicitly specified in the SDL by the deployer.

In this case I believe multiple bids by the provider are required.

Opinions?

thanks,

Shimpa

**Comment by ghost** [comment]
@shimpa1 there is an issue for that https://github.com/akash-network/support/issues/166

**Reply by shimpa1** [reply]
Oh awesome thank you!
